cmake/onnxruntime_unittests.cmake

```diff
@@ -428,7 +429,7 @@ endif()  # SingleUnitTestProject
 AddTest(
   TARGET onnxruntime_test_framework_session_without_environment_standalone
   SOURCES "${TEST_SRC_DIR}/framework/inference_session_without_environment/inference_session_without_environment_standalone_test.cc" "${TEST_SRC_DIR}/framework/test_main.cc"
-  LIBS  onnxruntime_test_utils ${ONNXRUNTIME_TEST_LIBS}
+  LIBS  onnxruntime_test_utils ${ONNXRUNTIME_TEST_LIBS} winml_adapter
```

Comment:
>winml_adapter [](start = 56, length = 13)

maybe move winml_adapter to ONNXRUNTIME_TEST_LIBS, and i dont think we will need to set it again below.

Reply:
removed.

---
In reply to: [370285810](https://github.com/microsoft/onnxruntime/pull/2891#discussion_r370285810) [](ancestors = 370285810)